cmd/cli/task_config/config_test.go

```diff
@@ -16,16 +16,16 @@ const (
 	testCfgPath = "test.yaml"
 )
 
-func createTestConfigFile(body string) error {
-	return ioutil.WriteFile(testCfgPath, []byte(body), 0600)
+func createTestConfigFile(p, body string) error {
```

Comment:
What it `p`?

Reply:
Also these tests changes looks completely out of the scope and only prevent proper understanding of the PR.